src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/arm/eltwise.cpp

```diff
@@ -127,6 +127,23 @@ const auto params_4D_bitwise_NOT_i32 = ::testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(smoke_CompareWithRefs_4D_Bitwise_NOT_i32, EltwiseLayerCPUTest, params_4D_bitwise_NOT_i32, EltwiseLayerCPUTest::getTestCaseName);
 
+const auto params_4D_int_jit = ::testing::Combine(
+        ::testing::Combine(
+                ::testing::ValuesIn(static_shapes_to_test_representation(inShapes_4D())),
+                ::testing::ValuesIn(eltwiseOpTypesBinInpJit()),
+                ::testing::ValuesIn(secondaryInputTypes()),
+                ::testing::ValuesIn(opTypes()),
+                ::testing::ValuesIn(netTypeJit()),
```

Comment:
Seems like `eltwiseOpTypesBinInpJit` and `netTypeJit` are used only inside Arm instance. In that case these functions shouls be declared and defined in `arm/eltwise.cpp`.
BTW, could you please clarify why we need to have separate test instance here? What it covers?